### ISSUE TITLE
NPM Config files appear busted for external dev

### ISSUE THREAD

**Issue by DeeWBee** [mParticle/mparticle-web-sdk#161]
Hello! 

I recently cloned this repo, and have been attempting to get setup. Unfortunately, I get stuck at the first step... Whenever I attempt to run `npm run setup`, I see the following errors (see attached screenshot). I am also attaching the `npm-debug.log` file. 

Furthermore, when attempting to run `npm run install`, I encounter Node version errors (`Expected version ">=8.0.0". Got "6.9.2"`). My Node manager (`NVM`) is unable to reconcile this. What Node manager is used within the SDK?

![Screen Shot 2021-08-03 at 1 39 46 PM](https://user-images.githubusercontent.com/13024101/128069037-6f1ef629-7478-4035-ac9c-4d7949166bbf.png)

[npm-debug.log](https://github.com/mParticle/mparticle-web-sdk/files/6926393/npm-debug.log)

In summary: the `setup` and `build` scripts are not functioning. The standard error messages claim that the repo setup is broken, but I find that hard to believe... what steps am I missing?

Best, 
Darius Bieganski
SportsEngine

**Comment by rmi22186** [comment]
We haven't received any feedback on this issue and so are closing.   Running `npm install`, `npm run test`, and `npm run build` all seem to be working in our CI both locally and on Github Actions.  @DeeWBee if there is still an issue please add further reproduction steps and reopen.  Thanks!
